jdi-light-vuetify-tests/src/test/resources/test.properties

```diff
@@ -23,4 +23,4 @@ log.level=INFO
 #remote.type=sauce | browserstack
 #driver.remote.url=http://localhost:4444/wd/hub
 #screenshot.strategy=on fail | on | off
-headless=true
+headless=false
```

Comment:
Please, do not forget to reverse changes in 'properties' files :)

Reply:
Done!